kafka/util/util.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import (
+	"log"
+	"os"
+)
+
+func GetEnv(envName string) string {
```

Comment:
I thought we said we didn't want this duplicated in two files?

Reply:
They are only in util.go file now